### PR TITLE
Fix build break.

### PR DESCRIPTION
**Pull request by Priya91**
cc @danmosemsft @weshaggard

### CODE REVIEW

File: src/packages.builds [hunk 1]
```diff
@@ -29,6 +29,12 @@
     <Project Include="..\pkg\Microsoft.NETCore.Targets\Microsoft.NETCore.Targets.builds">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
+    <Project Include="System.Net.Http\pkg\System.Net.Http.builds">
```
Comment (joperezr) [inline src/packages.builds]: Why wouldn't this just be automatically includded with:
```xml
     <Project Include="..\pkg\*\*.builds" Condition="'$(SkipManagedPackageBuild)' != 'true'">
       <AdditionalProperties>$(AdditionalProperties)</AdditionalProperties>
     </Project>
```
Do we want to build this even when SkipManagedPackageBuild is set to true? @weshaggard

**Comment by danmosemsft** [comment]
@Priya91 you can consider this shiproom approved. Any chance you could get it in today?

File: src/packages.builds (hunk 1 above)
Comment (weshaggard) [inline src/packages.builds]: This is for servicing so we only build specific packages.

**Comment by MattGal** [comment]
Chatted with @bartonjs, the merge conflict was trivial so I fixed it up to match what both parties wanted.  Will wait for CI success to merge.

**Comment by danmosemsft** [comment]
Thanks @MattGal !

**Pull request merged**